### PR TITLE
feat: Initial configuration for sentry.io

### PR DESCRIPTION
**Pull request by eitu5ami**
# Description

Initial integration with Sentry.

# Testing Instructions

By default, Sentry is disabled. In order to test it, you have to setup additional environment variables: `SENTRY_ENABLED` `SENTRY_DSN` and `SENTRY_ENVIRONMENT`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +83,9 @@
         "@0x/utils": "^6.5.3",
         "@0x/web3-wrapper": "^7.6.5",
         "@ethersproject/hdnode": "^5.2.0",
+        "@sentry/node": "^7.9.0",
+        "@sentry/tracing": "^7.9.0",
+        "@sentry/types": "^7.9.0",
```
Comment (kyu-c) [inline package.json:88]: nit: types should go under `devDependencies`

File: src/sentry.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            }),
+        ],
+        tracesSampler: (context: SamplingContext) => {
+            // Do not submit traces for health check.
+            //
+            if (context?.transactionContext?.name.includes(HEALTHCHECK_PATH)) {
+                return false;
+            }
+
+            return true;
```
Comment (kyu-c) [inline src/sentry.ts:33]: How expensive tracings are? Would it make sense to set this lower? (say 1%.. or 10%)?

**Review by kyu-c — APPROVED** [review]
🔥

File: src/runners/http_service_runner.ts [hunk 3]
```diff
@@ -71,6 +72,15 @@ export async function runHttpServiceAsync(
         logger.error(err);
     });
 
+    if (dependencies.hasSentry) {
+        const options: SentryOptions = {
+            app,
+            paths: [SRA_PATH, ORDERBOOK_PATH, META_TRANSACTION_PATH, SWAP_PATH],
```
Comment (idokleinman) [inline src/runners/http_service_runner.ts:78]: Is it ok that SWAP_PATH is duplicated by this runner init + the swap service runner init?

**Review by idokleinman — APPROVED** [review]

**Pull request merged**